doc/website/release-notes/iceoryx-unreleased.md

```diff
@@ -150,6 +152,7 @@
 
    delete soSmart; // <- not possible anymore
    ```
+
```

Comment:
I guess this superfluous line came in with the merge. In case you need to add further commits to this PR, please remove it but a commit just for this is nor necessary.

Reply:
No it is not superfluous I added it explicitly. I like to have a empty line between each entry of an enumeration.  When you take at the previous release notes you see it is consistent, also in this files with the entries above.

Reply:
Aaargh, it looked like there was already an empty line but it's the line with the backticks